modules/simple_forms_api/app/models/simple_forms_api/vha_10_10d.rb

```diff
@@ -18,17 +20,25 @@ def metadata
         'zipCode' => @data.dig('veteran', 'address', 'postal_code') || '00000',
         'source' => 'VA Platform Digital Forms',
         'docType' => @data['form_number'],
-        'businessLine' => 'CMP'
+        'businessLine' => 'CMP',
+        'ssn_or_tin' => @data.dig('veteran', 'ssn_or_tin')
       }
     end
 
+    def generate_secure_uuid
+      SecureRandom.uuid
+    end
+
```

Comment:
Since we only call it once, it might make sense to remove this wrapper around `SecureRandom.uuid` since it's only one line.

Reply:
I agree we don't have to worry about it being DRY in this case